Sources/Network/HTTPS/RestApiRequester.swift

```diff
@@ -16,15 +16,21 @@ public class RestApiRequester {
 }
 
 protocol Requester {
-    func makeRequest<T: HTTPClientCall>(call: T, completion: @escaping (HttpCallResult<T.ResponsePayload>) -> Void)
+    func makeRequest<T: HTTPClientCall>(
+            call: T,
+            completion: @escaping (HttpCallResult<T.ResponsePayload>) -> Void
+        )
 }
 
-extension RestApiRequester : Requester {
+extension RestApiRequester: Requester {
     private func completeURL(path: String) -> URL? {
-        .prefix(baseUrl.httpBasedUrl, pathComponents:[prefix, path])
+        .prefix(baseUrl.httpBasedUrl, pathComponents: [prefix, path])
     }
-    
-    public func makeRequest<T: HTTPClientCall>(call: T, completion: @escaping (HttpCallResult<T.ResponsePayload>) -> Void) {
+
+    public func makeRequest<T: HTTPClientCall>(
+            call: T,
+            completion: @escaping (HttpCallResult<T.ResponsePayload>) -> Void
```

Comment:
Single-indentation for wrapped function params in definition:

```suggestion
        call: T,
        completion: @escaping (HttpCallResult<T.ResponsePayload>) -> Void
```

Reply:
accepting